torch/fx/symbolic_trace.py

```diff
@@ -157,7 +174,7 @@ def proxy_placeholder(name: str):
                 args.append(proxy_placeholder('**' + next(names_iter)))
             root_fn = _patch_function(root_fn, len(args))
 
-        return root_fn, args
+        return fn_for_analysis, args
```

Comment:
This isn't correct: `fn_for_analysis` is just for analysis, if you return it here then we'll be tracing through a function that's been stripped of various things (wrappers, etc).

Can you please just revert the changes to this file and add `_rewrite_` to `rewriter.py`, and then call that within a `RewritingTracer` class also in that file?

Since this is super experimental I don't think we want the entire FX test surface to go through this code like it is right now

Reply:
These changes are still here but they should be reverted